be/src/pipeline/pipeline_fragment_context.cpp

```diff
@@ -294,6 +294,9 @@ Status PipelineFragmentContext::prepare(const doris::TPipelineFragmentParams& re
         if (request.__isset.load_job_id) {
             _runtime_state->set_load_job_id(request.load_job_id);
         }
+        if (request.__isset.llm_resources) {
```

Comment:
should do the job in query context constructor `**_get_or_create_query_ctx**` func